code/modules/economy/cash.dm

```diff
@@ -47,42 +47,43 @@
 		return list(icon_state)
 
 /obj/item/spacecash/bundle
-	name = "pile of thalers"
+	name = "pile of dollars"
 	icon_state = "spacecash1"
-	desc = "They are worth 0 Thalers."
+	desc = "It's worth 0 dollars."
 	worth = 0
 
 /obj/item/spacecash/bundle/Initialize()
 	. = ..()
 	update_icon()
 
 /obj/item/spacecash/bundle/getMoneyImages()
-	if(icon_state)
-		return list(icon_state)
 	. = list()
 	var/sum = src.worth
 	var/num = 0
 	for(var/i in denominations)
-		while(sum >= i && num < 50)
+		while(sum >= i && num < 10)
 			sum -= i
 			num++
 			. += "spacecash[i]"
-	if(num == 0) // Less than one thaler, let's just make it look like 1 for ease
+	if(num == 0) // Less than one dollar, let's just make it look like 1 for ease
 		. += "spacecash1"
 
 /obj/item/spacecash/bundle/on_update_icon()
 	cut_overlays()
 	var/list/images = src.getMoneyImages()
 
+	icon_state = images[1]	// if we don't do this, there will be an extra image
+	images -= images[1]
```

Comment:
why'll there be an extra image?

Reply:
It overlays the money over the current sprite. So, if you have 5 bucks in a stack it'll overlay 5 1-dollar bills over a picture of a 1-dollar bill, for a total of 6 bucks. Also has the (positive) side effect of not rendering single-bill stacks with rotation effects. 

I'll add some more comments to make this apparent.